msi/rust.wxs

```diff
@@ -192,9 +201,9 @@
                         <Shortcut Id="RustShell"
                                   Name="$(var.ProductName) Shell"
                                   Description="Opens Command Prompt with Rust tools directory added to the PATH"
-                                  Target="[SystemFolder]CMD.exe"
+                                  Target="[SHORTCUTTARGET]"
                                   Arguments="/K path [INSTALLDIR]bin;%PATH%"
-                                  WorkingDirectory="PersonalFolder">
+                                  WorkingDirectory="SHORTCUTWKDIR">
```

Comment:
I would love to set it to `%HOMEDRIVE%%HOMEPATH%` but unfortunately Windows Installer expands environment variables in the Working Directory during the install, see [here](https://msdn.microsoft.com/en-us/library/windows/desktop/aa371847.aspx#WkDir). So as far as I'm aware we can't do that. I only chose the system drive root because it isn't user specific.